spec/calendar_assistant/cli/features/location_spec.rb

```diff
@@ -6,11 +6,17 @@
   let(:filename) { temp_file.path }
 
   before(:each) do
-    event_list_factory(file: filename, time_zone: "Pacific/Fiji") do
+    event_list_factory(file: filename, calendar_id: "primary", time_zone: "Pacific/Fiji") do
```

Comment:
Can we change this to use the constant `DEFAULT_CALENDAR_ID`?

Reply:
Fixed